src/screens/SignTransactionSheet.tsx

```diff
@@ -950,6 +950,7 @@ export const SignTransactionSheet = () => {
                     }
                     newShadows
                     onPress={submitFn}
+                    disabled={!walletBalance?.isLoaded || !selectedGasFee.gasFee.estimatedFee}
```

Comment:
are there any other cases where this should be disabled? If so, I think we should move this out to a functional check.

What should happen if the tx simulation hasn't resolved yet? Should we disable the button during that?

Reply:
i think people should be able to transact if they choose to without waiting for sim since it's not vital